stores/locationStore.ts

```diff
@@ -0,0 +1,36 @@
+import { defineStore } from 'pinia'
+import { useModalStore } from './modalStore'
+
+export const useLocationStore = defineStore({
+    id: 'location-store',
+    state: () => ({
+        healthcareProfessionalName: null,
+        specialty: null,
+        facilityName: null,
+        spokenLanguages: null,
+        address: null,
+        hours: null,
+        website: null,
+        phone: null,
+        position: { lat: 35.699615, lng: 139.545596 }
+    }),
+    actions: {
+        setLocationDetails(location: { healthcareProfessionalName: string;
```

Comment:
would it make sense to have this be a typescript type?

Reply:
Like I mentioned in the PR description, this file is probably temporary and will be rewritten as we're connecting this to the backend. Feel free to make changes!